src/mod/server.mod/server.c

```diff
@@ -121,6 +121,15 @@ static void msgq_clear(struct msgq_head *qh);
 static int stack_limit;
 static char *realservername;
 
+static int sasl = 0;
+static int account_notify = 0; //TODO Remove after testing
+static int foober = 1;
+
+static char sasl_mechanism[25];
```

Comment:
static char sasl_mechanism[25] = 0;

default to 0, to match the config setting.

Reply:
fixed. the code still treated sasl_mechanism as string not enum.